src/lightning/data/streaming/functions.py

```diff
@@ -168,7 +177,7 @@ def optimize(
         fast_dev_run: Whether to use process only a sub part of the inputs
         num_nodes: When doing remote execution, the number of nodes to use.
         machine: When doing remote execution, the machine to use.
-        num_downloaders: The number of downloaders per worker.
+        num_downloaders: The number of downloaders to use.
```

Comment:
Maybe "number of downloaders per machine" would be clearer?

Reply:
Reverted the change. This would be removed in further iteration by using some kinda of auto-tuning.